pkgs/development/libraries/rocsolver/default.nix

```diff
@@ -0,0 +1,104 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, writeScript
+, cmake
+, rocm-cmake
+, rocm-runtime
+, rocm-device-libs
+, rocm-comgr
+, rocblas
+, hip
+, fmt
+, gtest ? null
+, gbenchmark ? null
+, buildTests ? false
+, buildBenchmarks ? false
+, gpuTargets ? null # gpuTargets = [ "gfx803" "gfx900" "gfx906:xnack-" ]
```

Comment:
This should be checked

Reply:
Impossible to do in the nix sandbox in a pure fashion.
It's why #200757 exists.